### ISSUE TITLE
Changing static parts of template is not reflected in compiled app

### ISSUE THREAD

**Issue by vasily-kirichenko** [intellifactory/websharper.ui.next#62]
* change any static text
* compile
* open the page in the browser
* static parts of the page is not changed

**Comment by Tarmil** [comment]
Yeah, unfortunately there isn't much we can do about this. The F# MSBuild task decides whether actual compilation is needed based on whether the F# source files have been modified, but if you only modify the HTML file then it thinks there are no changes and doesn't run the compiler. Doing a Ctrl+S on an F# file before compiling is enough to trick it into running.

**Comment by vasily-kirichenko** [comment]
I believe the TP should trigger the compiler after being invalidated by file system watcher. Isn't it the case?

**Comment by Tarmil** [comment]
I think invalidation only affects the code service, which is separate from the build task.

**Comment by smoothdeveloper** [comment]
I've noticed that with type providers in general, should we open an issue on [Microsoft/visualfsharp](https://github.com/Microsoft/visualfsharp/) so that the type provider infrastructure gets ability to surface additional dependencies for msbuild tasks?

**Comment by Tarmil** [comment]
It's quite surprising that there is no msbuild action (like Compile, Content, None, Resource, etc) for items like this, that are not compiled but on which compilation depends.
